cosmo_tester/resources/terraform/scripts/osx-cli-test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+MANAGER_BLUEPRINTS_PATH="/usr/local/opt/cfy/cloudify-manager-blueprints"
+MANAGER_BLUEPRINT_PATH="${MANAGER_BLUEPRINTS_PATH}/simple-manager-blueprint.yaml"
+INPUTS_FILE_PATH="/tmp/bootstrap_inputs.yaml"
+
+CLI_PACKAGE_URL=$1
+PRIVATE_KEY_PATH=$2
+PUBLIC_IP=$3
+PRIVATE_IP=$4
+MANAGER_USER=$5
+
+echo "Installing Cloudify's CLI..."
+
+
+curl ${CLI_PACKAGE_URL} -o /tmp/package.pkg
+echo $MACINCLOUD_PASSWORD | sudo -S installer -pkg /tmp/package.pkg -target /
+
+set -e
+
+echo "Creating inputs file.."
+echo "public_ip: ${PUBLIC_IP}
+private_ip: ${PRIVATE_IP}
+ssh_user: ${MANAGER_USER}
+ssh_key_filename: ${PRIVATE_KEY_PATH}
+admin_username: admin
+admin_password: admin" > ${INPUTS_FILE_PATH}
+cat ${INPUTS_FILE_PATH}
+
+echo "Setting permissions for private key file: ${PRIVATE_KEY_PATH}"
+#chmod 400 ${PRIVATE_KEY_PATH}
```

Comment:
There's not much point keeping commented code when using version control, so this line should probably be removed if it's not required any more (and then the preceding echo should go too as it'll be lying).

Reply:
You are right. I forgot about this line.